library/turtle.po

```diff
@@ -67,18 +80,24 @@ msgid ""
 "programmer to use all the commands, classes and methods interactively when "
 "using the module from within IDLE run with the ``-n`` switch."
 msgstr ""
+"La librairie essaye de garder les avantages de l'ancien module et d'être "
+"(presque) 100% compatible avec celui-ci. Cela permet à l'apprenti développeur "
```

Comment:
Il manque la suite 😁

Reply:
translated forgotten part in last commit.